target/linux/ath79/image/generic.mk

```diff
@@ -421,6 +421,20 @@ define Device/netgear_wndr3x00
   DEVICE_PACKAGES := kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport kmod-leds-reset kmod-owl-loader
 endef
 
+define Device/netgear_ex7300
+  ATH_SOC := qca9558
+  IMAGE/default := append-kernel | pad-offset $$$$(BLOCKSIZE) 64 | netgear-rootfs | pad-rootfs
+  $(Device/netgear_ath79)
+  DEVICE_TITLE := NETGEAR EX6400 or EX7300
+  NETGEAR_KERNEL_MAGIC := 0x27051956
+  NETGEAR_BOARD_ID := EX7300series
+  NETGEAR_HW_ID := 29765104+16+0+128
+  IMAGE_SIZE := 15552k
+  SUPPORTED_DEVICES += ex7300
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca99x0
```

Comment:
61b5b4971e7d6aab6b0695997c3b5aaf73c53b5f made ath10k-ct the default.

kmod-ath10k-ct ath10k-firmware-qca99x0-ct

Reply:
Oops. Will fix.